Project/Models/bi_lstm_model.py

```diff
@@ -0,0 +1,115 @@
+# import libraries
+import os
+import sys
+import nltk
+
+try:
+    from nltk.corpus import stopwords
+except:
+    nltk.download('stopwords')
+    from nltk.corpus import stopwords
+from keras.datasets import imdb
+from tensorflow import keras
+from tensorflow.keras import layers
+
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Dense, Embedding
+from tensorflow.keras.layers import LSTM
+
+from models.ml_utilities import *
+
+from models.ml_model_template import MLModelTemplate
+
+
+# define class
+class BiLSTMModel(MLModelTemplate):
+
+    def __init__(self):
+        self.__max_features = 20000  # Only consider the top 20k words
+        self.__maxlen = 200  # Only consider the first 200 words of each movie review
+
+    #Load the IMDB movie review sentiment data
+    def load_data(self):
+        (self.__x_train, self.__y_train), (self.__x_val, self.__y_val) = keras.datasets.imdb.load_data(num_words=self.__max_features)
+        self.__x_train = keras.preprocessing.sequence.pad_sequences(self.__x_train, maxlen=self.__maxlen)
+        self.__x_val = keras.preprocessing.sequence.pad_sequences(self.__x_val, maxlen=self.__maxlen)
+
+    # Method for creating ML->RNN->Bi_LSTM model
+    def create_model(self):
+        print("Creating model...")
+        # Input for variable-length sequences of integers
+        inputs = keras.Input(shape=(None,), dtype="int32")
+        # Embed each integer in a 128-dimensional vector
+        x = layers.Embedding(self.__max_features, 128)(inputs)
+        # Add 2 bidirectional LSTMs
+        x = layers.Bidirectional(layers.LSTM(64, return_sequences=True))(x)
+        x = layers.Bidirectional(layers.LSTM(64))(x)
+        # Add a classifier
+        outputs = layers.Dense(1, activation="sigmoid")(x)
+        model = keras.Model(inputs, outputs)
+        print("Model created...!!!")
+        return model
+
+    # Method for training the model
+    def train_model(self, model):
+        print("Training Model... Please wait!")
+        model.compile("adam", "binary_crossentropy", metrics=["accuracy"])

```

Comment:
I see you have used different keras syntax. Can we move compile model code to create_model to maintain the uniformity with other code file.